design-patterns/gc-intro/samples/gc-intro-image.html

```diff
@@ -0,0 +1,11 @@
+<div class="bg-center bg-cover" data-bgimg-srcset="img/1x1.png 991w, img/ip-cover-image-1200x726.jpg 992w">
+	<div class="container">
+		<div class="row mrgn-bttm-md">
+			<div class="col-md-7">
+				<h1 property="name" id="wb-cont" dir="ltr">{{ page.title }}</h1>
```

Comment:
Remove the attribute `dir` as by default it is left to right. When user use special text direction, they will apply the lang attribute too on the element.

```suggestion
				<h1 property="name" id="wb-cont">{{ page.title }}</h1>
```